### PR TITLE
Use last vscode release

### PR DESCRIPTION
**Pull request by CGNonofr**
Here's the diff in term of what is taken from VSCode: https://github.com/microsoft/vscode/compare/af28b32d7e553898b2a91af498b1fb666fdebe0c..1.85.2

What is especially required is: https://github.com/microsoft/vscode/pull/200711

I've also backported this fix: https://github.com/microsoft/vscode/pull/201406 even if I've failed to see any improvement on it

### CODE REVIEW

File: docs/vscode_monaco_upgrade.md [hunk 1]
```diff
@@ -2,22 +2,22 @@
 
 ## Preparation
 
-- It is assumed `monaco-vscode-api`, `vscode` and `monaco-editor` repos are cloned locally on the same directory level. `monaco-editor` is not required, but helpful
+- It is assumed `monaco-vscode-api` and `vscode` and `monaco-editor`.
```
Comment (kaisalmen) [inline docs/vscode_monaco_upgrade.md:5]: The sentence is not complete. 🙂

**Review by kaisalmen — APPROVED** [review]
Do you want to update the upgrade doc? Otherwise LGTM

File: docs/vscode_monaco_upgrade.md (hunk 1 above)
Comment (CGNonofr) [reply docs/vscode_monaco_upgrade.md:5]: Wtf did I do? 😨 will update

Comment (CGNonofr) [reply docs/vscode_monaco_upgrade.md]: Fixed! also added 2 new commits to synchronize library versions

File: scripts/update-vscode-dependencies.ts [hunk 2]
```diff
@@ -34,11 +44,28 @@ async function updateVSCodeDependencies () {
       apiPackageJson.overrides[dependency] = vsCodePackageJson.dependencies[dependency]
     }
   }
+
+  console.debug('Updating monaco-vscode-api ncurc to prevent VSCode dependencies from being updated by ncu')
+  const ncuRcPath = path.join(__dirname, '../.ncurc.json')
+  const ncuRc = await readJson(ncuRcPath)
+  ncuRc.reject = [
+    '@types/node',
```
Comment (kaisalmen) [inline scripts/update-vscode-dependencies.ts:52]: Aren't those already contained in `.ncurc.json`?

**Review by kaisalmen — APPROVED** [review]
My comment is non-blocking. I just had the impression the `ncuRc.reject` array does already contain what you add.

File: scripts/update-vscode-dependencies.ts (hunk 2 above)
Comment (CGNonofr) [reply scripts/update-vscode-dependencies.ts:52]: Yes, and I want to keep them! it I don't put theme here, they'll be removed

**Pull request merged**